src/domUpdates.js

```diff
@@ -5,36 +5,54 @@ const domUpdates = {
   },
 
   showCustomerDashboard() {
+    this.hideAll();
+
     let customerDashboard = document.querySelector('.customer-dashboard');
     customerDashboard.classList.remove('hidden');
   },
 
   showManagerDashboard() {
+    this.hideAll();
+
     let managerDashboard = document.querySelector('.manager-dashboard');
     managerDashboard.classList.remove('hidden');
   },
 
   showLoginDashboard() {
+    this.hideAll();
+
     let loginDashboard = document.querySelector('.login-dashboard');
     loginDashboard.classList.remove('hidden');
   },
 
   checkPassword() {
     let passwordInput = document.getElementById('password-input');
-    return (passwordInput.value === 'overlook2020')
+    return(passwordInput.value === 'overlook2020');
```

Comment:
I'm doing the same thing but left this commented out to make testing a little easier